src/components/ProductsSlider/ProductsSlider.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { FC, memo, useState } from 'react';
+import { Product } from '../../types/Product';
+import { ProductItem } from '../ProductItem';
+import arrowLeft from '../../assets/svg/arrowLeft.svg';
+import arrowRigth from '../../assets/svg/arrowRigth.svg';
+import './products-slider.scss';
+
+type Props = {
+  title: string;
+  products: Product[];
+};
+
+export const ProductsSlider: FC<Props> = memo(({ title, products }) => {
+  const visibleCount = 4;
```

Comment:
Move out of conponent. No need need to create same variable on each render

Reply:
good point